pkgs/servers/sunshine/default.nix

```diff
@@ -163,6 +163,10 @@ stdenv'.mkDerivation rec {
       --replace '@PROJECT_NAME@' 'Sunshine' \
       --replace '@PROJECT_DESCRIPTION@' 'Self-hosted game stream host for Moonlight' \
       --replace '/usr/bin/env systemctl start --u sunshine' 'sunshine'
+
+    substituteInPlace packaging/linux/sunshine.service.in \
+      --replace '@PROJECT_DESCRIPTION@' 'Self-hosted game stream host for Moonlight' \
```

Comment:
Believe `--replace` is deprecated -- suggest using `--replace-fail` or `--subst-var-by` (working on changing them to a mix of these in the next update PR).

Reply:
https://github.com/NixOS/nixpkgs/pull/296747 see here for examples

Reply:
I changed --replace to --subst-var-by in the latest commit.